CHANGELOG.md

```diff
@@ -7,6 +7,12 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+
+## 12.21.0
+* Enhancing layout footer component so that it can support links that accept 
+  a collection of anchor attributes. The new feature was introduced to allow
+  developers to support the target attribute for opening new windows.(PR #664)
```

Comment:
This feels a bit verbose, could we go down just "Layout footer component now accepts an attributes hash for each link element"

Reply:
Done